lforms-converter.js

```diff
@@ -398,6 +398,9 @@ function doSkipLogic(root) {
 
   traverseItems(root, function(item, ancestors) {
     if(item.skipLogic) {
+      if (item.skipLogic.condition != undefined && item.skipLogic.condition.trim() === "") {
+        delete item.skipLogic;
+      } else {
       // This is target item. Parse 'condition' to look for source item
       var tokens = item.skipLogic.condition.split('=');
```

Comment:
Here's an example of a form that will not get through LForms Converter:
https://cde.nlm.nih.gov/formView?tinyId=mktzfTiYx